### PR TITLE
option to not display Suggestions for improvement

### PR DESCRIPTION
**Pull request by pnpk**
Thank you for publishing such a wonderful application.

I've created a pull request because I wanted an option to not display 'Suggestions for improvement'.

By adding `display-suggestions: false` to the arguments, I tried to prevent 'Suggestions for improvement' from being displayed.

I hope you will consider adopting it.
Thank you in advance.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -122,6 +122,8 @@ async function Run(): Promise<void>
     ThrowIfParametersMissing()
     ThrowIfNotSupportedEvent()
 
+    const displaySuggestions = core.getInput('display-suggestions') != 'false';
```
Comment (akiojin) [inline src/main.ts:125]: @pnpk 
How about using core.getBooleanInput?

```
core.getBooleanInput('display-suggestions')
```

File: action.yml [hunk 2]
```diff
@@ -47,6 +47,11 @@ inputs:
       If not optimized, all differences from the base branch are reviewed.
     required: false
     default: true
+  display-suggestions:
+    description: >
+      You can specify whether to display improvement suggestions.
+      ex) false
+    required: false
```
Comment (akiojin) [inline action.yml:54]: @pnpk 
Would it be easier to understand by setting a default value?

```
default: true
```

**Comment by akiojin** [comment]
@pnpk 
Thank you for your suggestion and for your
We have included your review in our website.

File: action.yml (hunk 2 above)
Comment (pnpk) [reply action.yml]: Certainly, that is kinder.

**Review by akiojin — APPROVED** [review]

**Pull request merged**